core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -2864,6 +2865,26 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     val partition2 = new TopicPartition("elect-preferred-leaders-topic-2", 0)
     createTopicWithAssignment(partition2.topic, Map[Int, Seq[Int]](partition2.partition -> prefer0))
 
+    var prior1 = brokers.head.metadataCache.getPartitionLeaderEndpoint(partition1.topic, partition1.partition(), listenerName).get.id()
+    var prior2 = brokers.head.metadataCache.getPartitionLeaderEndpoint(partition2.topic, partition2.partition(), listenerName).get.id()
+
+    def sleepMillisToPropagateMetadata(durationMs: Long): Unit = {
+      TimeUnit.MILLISECONDS.sleep(100)
```

Comment:
In the following `TestUtils.waitUntilTrue`, it already do sleep if the condition is not matched. Do we need `TimeUnit.MILLISECONDS.sleep(100)` here?

Reply:
Hello @FrankYang0529 , I think we can move `TimeUnit.MILLISECONDS.sleep(100) here`.
Thank you for pointing out!